applications/print_array_elements.py

```diff
@@ -29,12 +29,33 @@
 
     Example:
 
+    First get the array element list
```

Comment:
Here you didn't add `get ... from DB`, was that a conscious decision?

Reply:
No, this was probably a lapse of memory. I will add it compatible to the previous case

Reply:
There is actually a difference between getting and adding a file to DB. In getting a file we don't need to indicate which db it is, we just search through all the available dbs, hence I will add here "from the DBs". On the other hand, in adding a file to DB  we have to indicate which DB, hence I used there "to a DB"